File: DrivingNote/ClientApp/src/Models/UserInformation.ts
import { Cell } from "./Cell";

export default class UserInformation {
  Name: string;
  LastName: string;
  Email: string;
  Sport: string;
  AccountNumber: string;
  TableInfomation: Cell[];

  constructor(name: string, lastname: string, email: string, sport: string, accountNumber: string, tableInfomation: Cell[]) {
    this.Name = name;
    this.LastName = lastname;
    this.Email = email;
    this.Sport = sport;
    this.AccountNumber = accountNumber;
    this.TableInfomation = tableInfomation;
  }
}

File: DrivingNote/Models/UserInformation.cs
using System.Collections.Generic;

namespace DrivingNote.Models
{
    public class UserInformation
    {
        public string Name { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Sport { get; set; }
        public string AccountNumber { get; set; }
        public List<TableInfomation> TableInfomation { get; set; }
    }
}
